.circleci/config.yml

```diff
@@ -33,11 +33,29 @@ jobs:
       - run: yarn run gen-snapshot-tests
       - run: yarn run build:cjs
       - run: yarn run build:es6
-      - run: yarn run jest --coverage
+      - run: yarn run jest --coverage --runInBand
       - run: yarn run codecov
+  vite:
+    docker:
+      - image: circleci/node:8-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-deps-{{ checksum "yarn.lock" }}
+            - npm-deps-
+      - run: yarn install
+      - save_cache:
+          key: npm-deps-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run: yarn run build:cjs
+      - run: yarn run build:es6
```

Comment:
We have to build both b/c the node runner still needs to be able to import `@khanacademy/wonder-blocks-foo` without having to compile which is why we're building the commonjs modules.  The es6 modules are being built because it's easier for `vite-server` to serve those modules to whatever browser selenium is controlling.